stdlib/builtin/module.rbs

```diff
@@ -1,248 +1,1069 @@
+  # optional parameter is `false`, the methods of any ancestors are not included.
+  #
   def public_instance_methods: (?bool include_super) -> ::Array[Symbol]
 
-  def public_method_defined?: (Symbol | String arg0) -> bool
-
+  # Returns `true` if the named public method is defined by *mod*.  If *inherit*
+  # is set, the lookup will also search *mod*'s ancestors. String arguments are
+  # converted to symbols.
+  #
+  #     module A
+  #       def method1()  end
+  #     end
+  #     class B
+  #       protected
+  #       def method2()  end
+  #     end
+  #     class C < B
+  #       include A
+  #       def method3()  end
+  #     end
+  #
+  #     A.method_defined? :method1                 #=> true
+  #     C.public_method_defined? "method1"         #=> true
+  #     C.public_method_defined? "method1", true   #=> true
+  #     C.public_method_defined? "method1", false  #=> true
+  #     C.public_method_defined? "method2"         #=> false
+  #     C.method_defined? "method2"                #=> true
+  #
+  def public_method_defined?: (Symbol | String arg0, ?bool inherit) -> bool
+
+  # Refine *mod* in the receiver.
+  #
+  # Returns a module, where refined methods are defined.
+  #
   def refine: (Class arg0) { (untyped arg0) -> untyped } -> self
 
+  # Removes the definition of the *sym*, returning that constant's value.
+  #
+  #     class Dummy
+  #       @@var = 99
+  #       puts @@var
+  #       remove_class_variable(:@@var)
+  #       p(defined? @@var)
+  #     end
+  #
+  # *produces:*
+  #
+  #     99
+  #     nil
+  #
   def remove_class_variable: (Symbol arg0) -> untyped
 
+  # Removes the definition of the given constant, returning that constant's
+  # previous value.  If that constant referred to a module, this will not change
+  # that module's name and can lead to confusion.
+  #
   def remove_const: (Symbol arg0) -> untyped
 
+  # Removes the method identified by *symbol* from the current class. For an
+  # example, see Module#undef_method. String arguments are converted to symbols.
+  #
   def remove_method: (Symbol | String arg0) -> self
 
-  # Returns `true` if *mod* is a singleton class or `false` if it is an
-  # ordinary class or module.
-  # 
-  # ```ruby
-  # class C
-  # end
-  # C.singleton_class?                  #=> false
-  # C.singleton_class.singleton_class?  #=> true
-  # ```
-  def `singleton_class?`: () -> bool
-
+  # Returns `true` if *mod* is a singleton class or `false` if it is an ordinary
+  # class or module.
+  #
+  #     class C
+  #     end
+  #     C.singleton_class?                  #=> false
+  #     C.singleton_class.singleton_class?  #=> true
+  #
+  def singleton_class?: () -> bool
+
+  # Returns a string representing this module or class. For basic classes and
+  # modules, this is the name. For singletons, we show information on the thing
+  # we're attached to as well.
+  #
   def to_s: () -> String
 
   def undefMethod: (Symbol | String arg0) -> self
```

Comment:
I asked in the Sorbet Slack about this, but I think they accidentally typo'd this and it should be `undef_method`?

Reply:
Good find! It should be `undef_method`. I think you can push another commit to fix it in this PR.